Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -40,6 +73,24 @@ class CodingOperation<ResponceType : Codable, OptionsType : DirectionsOptions >
         }
     }
     
+    #if !os(Linux)
+    private func interpolate(route: Route) -> [CLLocationCoordinate2D?] {
+        guard route.expectedTravelTime > 0, let polyline = route.shape,
+              let firstCoordinate = polyline.coordinates.first else { return [] }
+        
+        var distanceAway: CLLocationDistance = 0
+        let distance = route.distance/route.expectedTravelTime
```

Comment:
It's a bit confusing to call `distance / time` a distance. Could you rename it to `averageSpeed` or `distancePerTick`?
And if you add `timeInterval` as an argument of this function, then you can have `distanceAway += speed * timeInterval` in the cycle below.